wormhole/wormhole.go

```diff
@@ -90,6 +97,14 @@ func (c *Client) wordCount() int {
 }
 
 func (c *Client) relayURL() internal.SimpleURL {
+	if c.TransitRelayAddress != "" {
```

Comment:
I think you're noticing that this function doesn't return an error but we need to call some functions that do. In the other cases we've used `MustNewSimpleURL` to avoid dealing with the error scenario and I think we just need to do the same here. Currently, the call to `errors.New` returns an error that's not being used 

I think this whole thing would probably look in nicer as a switch statement, even though there are only 3 branches. I'm imaging something like this (note: the default case is [taken from out of the scope if this diff](https://github.com/LeastAuthority/wormhole-william/blob/fdb37350b7952fec97e048d2eeae7f6656d32628/wormhole/wormhole.go#L111)):
```golang
switch {
case c.TransitRelayAddress != "":
  return internal.MustNewSimpleURL(c.TransitRelayAddress)
case c.TransitRelayAddress != "":
    return internal.MustNewSimpleURL(c.TransitRelayURL)
default:
  return internal.MustNewSimpleURL(DefaultTransitRelayURL)
}
```